evaluation/mint/run_infer.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+SUBSET=$1
+EVAL_LIMIT=$2
```

Comment:
Can you add default value for SUBSET and EVAL_LIMIT, you can refer to the `run_infer.sh` for swe bench

Reply:
I've fixed it, can you have a look to see if it is what you're expecting?